modules/telco-ran-ptp-operator.adoc

```diff
@@ -7,17 +7,17 @@
 = PTP Operator
 
 New in this release::
-* https://issues.redhat.com/browse/CNF-12504[Enable 2 Port Ordinary Clock Use Case for IA]
-* https://issues.redhat.com/browse/CNF-13189[CNF-13189: Remove the internal API and sidecar support]
-* https://issues.redhat.com/browse/CNF-16651[CNF-16651: 3 Card WPC T-G]
+* Dual-port NIC for PTP ordinary clock is enabled.
+* The PTP events REST API v1 and events consumer application sidecar support are removed.
+* A maximum of 3 Westport channel NIC configurations is now supported for T-GM.
 
 Description::
 Configure PTP in cluster nodes with `PTPConfig` CRs for the RAN DU use case with features like Grandmaster clock (T-GM) support using GPS, ordinary clock (OC), boundary clocks (T-BC), dual boundary clocks, high availability (HA), and optional fast event notification over HTTP.
 PTP ensures precise timing and reliability in the RAN environment.
 
 Limits and requirements::
-* Limited to two boundary clocks for nodes with dual NICs and HA
-* Limited to two Westport channel NIC configurations for T-GM
+* Limited to 2 boundary clocks for nodes with dual NICs and HA
+* Limited to 3 Westport channel NIC configurations for T-GM
```

Comment:
@slovern did you intend to change the value from 2 to 3 here?

Reply:
yes. the limit is now 3, per line 12 above.